include/IGameLogic.hpp

```diff
@@ -0,0 +1,24 @@
+/*
+** EPITECH PROJECT, 2023
+** Visual Studio Live Share (Workspace)
+** File description:
+** ISystem
+*/
+
+#pragma once
+
+#include <vector>
+#include "IEntity.hpp"
+#include "IEventManager.hpp"
+#include "ISystem.hpp"
+
+namespace Arcade {
+    namespace Game {
+        class IGameLogic : ISystem {
+          public:
+            virtual ~IGameLogic() = default;
+            virtual void run(
```

Comment:
Not modifying sytem instance but the entities, so it's const, and same for both childs

Reply:
yes at least i think its better to let everyone to have attributes and modify it in game and graph systems